h/static/scripts/group-forms/components/test/CreateGroupForm-test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from 'enzyme';
+
+import CreateGroupForm from '../CreateGroupForm';
+
+describe('CreateGroupForm', () => {
+  const createWrapper = () => mount(<CreateGroupForm />);
+
+  [
+    { counter: 'name', limit: 25 },
+    { counter: 'description', limit: 250 },
+  ].forEach(({ counter, limit }) => {
+    describe(`${counter} character counter`, () => {
+      it(`initially renders 0/${limit}`, () => {
+        const wrapper = createWrapper();
+        const counterNode = wrapper.find(
+          `[data-testid="character-counter-${counter}"]`,
```

Comment:
Just a heads up, if you use `data-testid` with component library elements (eg. `<Button data-testid="foo" ...>`) a query like this can unintentionally end up matching multiple nodes, because the `data-testid` will be present on both the `Button` component and the `button` it renders. The solution is to specify the element type (`wrapper.find('button[data-testid="..."]')`)

Reply:
https://hypothes-is.slack.com/archives/C1M8NH76X/p1720720330967099?thread_ts=1720717859.036639&cid=C1M8NH76X ;)